libs/ui/src/app_base.tsx

```diff
@@ -93,6 +94,7 @@ export function AppBase(props: AppBaseProps): JSX.Element {
         screenType={screenType}
         sizeMode={sizeMode}
       >
+        <KeyboardShortcutHandlers />
```

Comment:
Re: the previous thread -- this isn't guaranteed to render within a `UiStringsContextProvider`, so better to keep rendering it there for now